docs/source/image_dataset_script.mdx

```diff
@@ -0,0 +1,245 @@
+...             "train": "https://link-to-breakfast-foods-train.txt", 
+...             "validation": "https://link-to-breakfast-foods-validation.txt"
+...         },
+...      ,
+...     Food101Config(
+...         name="dinner",
+...         description="Food types commonly eaten during dinner.",
+...         data_url="https://link-to-dinner-foods.zip",
+...         metadata_urls={
+...             "train": "https://link-to-dinner-foods-train.txt", 
+...             "validation": "https://link-to-dinner-foods-validation.txt"
+...         },
+...     )...
+... ]
+```
+
+Now if users want to load the `breakfast` configuration, they can use the configuration name:
+
+```py
+>>> from datasets import load_dataset
+>>> ds = load_dataset("food101", "breakfast", split="train")
+```
+
+## Add dataset metadata
+
+Adding information about your dataset is useful for users to learn more about it. This information is stored in the [`DatasetInfo`] class which is returned by the `info` method. Users can access this information by:
+
+```py
+>>> from datasets import load_dataset_builder
+>>> ds_builder = load_dataset_builder("food101")
+>>> ds_builder.info
+```
+
+There is a lot of information you can specify about your dataset, but some important ones to include are:
+
+1. `description` provides a concise description of the dataset.
+2. `features` specify the dataset column types. Since you're creating an image loading script, you'll need to include the [`Image`] feature.
+3. `supervised_keys` specify the input feature and label.
+4. `homepage` provides a link to the dataset homepage.
+5. `citation` is a BibTeX citation of the dataset.
+6. `license` states the dataset's license.
+
+<Tip>
+
+You'll notice a lot of the dataset information is defined earlier in the loading script which makes it easier to read. There are also other [`~Datasets.Features`] you can input, so be sure to check out the full list for more details.
+
+</Tip>
+
+```py
+>>> def _info(self):
+...     return datasets.DatasetInfo(
+...         description=_DESCRIPTION,
+...         features=datasets.Features(
+...             {
+...                 "image": datasets.Image(),
+...                 "label": datasets.ClassLabel(names=_NAMES),
+...             }
+...         ),
+...         supervised_keys=("image", "label"),
+...         homepage=_HOMEPAGE,
+...         citation=_CITATION,
+...         license=_LICENSE,
+...         task_templates=[ImageClassification(image_column="image", label_column="label")],
+...     )
+```
+
+## Download and define the dataset splits
+
+Now that you've added some information about your dataset, the next step is to download the dataset and generate the splits.
+
+1. Use the [`DownloadManager.download`] method to download the dataset and any other metadata you'd like to associate with it. This function accepts a URL or a list or dictionary of URLs. In the Food-101 loading script, you'll notice again the URLs are defined earlier in the script.
```

Comment:
Maybe here we can encourage users to upload their images to the Hub, instead of hosting them somewhere and pass the URL
```suggestion
1. Use the [`DownloadManager.download`] method to download the dataset and any other metadata you'd like to associate with it. This function accepts a name to a file inside the dataset repository (e.g. in the `data/` folder), or a URL to a file hosted elsewhere. It also supports a list or dictionary of file names or URLs. In the Food-101 loading script, you'll notice again the URLs are defined earlier in the script.
```